### ISSUE TITLE
"Edit Application" menu not launching

### ISSUE THREAD

**Issue by mukaschultze** [adhec/dittoMenuKDE#2]
Clicking on the `Edit Application` menu doesn't do anything and fails with:

```
.../org.kde.plasma.dittomenu/contents/code/tools.js:173: TypeError: Cannot call method 'substr' of undefined
```

![2021-09-05_14-42](https://user-images.githubusercontent.com/13923364/132136322-869bd6f4-fa48-4eb9-b768-3f5bf206f4ac.png)

Using `KDE Plasma 5.22.5` on Manjaro

**Comment by Setherz6104** [comment]
I'm having the same issue.

**Comment by Herreramartos** [comment]
Same in arch btw

**Comment by adhec** [comment]
Thanks for reporting, the script was fixed in #10
